### DISCUSSION TITLE
Is there a way to set css-loader options?

### DISCUSSION THREAD

**Discussion by vhpoet** [vercel/next.js#14169] [Help]
I'm not sure why [this](https://github.com/vercel/next.js/issues/10339) got closed, I might be missing something here. Is there a way to set the `localIdentName`?

**Comment by Tylerian** [comment]
Unfortunately it's hardcoded for now although it might change in the future judging [relevant src comments](https://github.com/vercel/next.js/blob/b88f20c90bf4659b8ad5cb2a27956005eac2c7e8/packages/next/build/webpack/config/blocks/css/loaders/modules.ts#L36)
